### ISSUE TITLE
Tutorial to setup from scratch

### ISSUE THREAD

**Issue by ReneTC** [dbt-labs/jaffle-shop-template#30]
Repo says the repo is for learning. 

But for a new user, the code-base is kinda a lot to grasp. Should we add a step-step tutorial on how to  setup this on a blank Meltano project?

**Comment by gwenwindflower** [comment]
definitely, there are a lot more resources coming around this 💗 still a bit of a WIP
